modules/Logger.js

```diff
@@ -0,0 +1,169 @@
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define(['underscore', 'backbone', './ServiceCell'], factory);
+  } else if (typeof exports === 'object') {
+    module.exports = factory(require('underscore'), require('backbone'), require('./ServiceCell'));
+  } else {
+    root.Torso = root.Torso || {};
+    root.Torso.Logger = factory(root._, root.Backbone, root.Torso.ServiceCell);
+  }
+}(this, function(_, Backbone, ServiceCell) {
+  'use strict';
+
+  var Logger = ServiceCell.extend({ 
+
+    log : {}, 
+    sessions: [],
+    unclosed: {},
+    _isClosed: true,
+    _sessionStart: null,
+    _sessionEnd: null,
+    _sessionID: null,
+
+    /**
+    * saves information about event and wraps start and end signals in 'session' 
+    * @ param {JSON object} contains tracking information (including state: 'start' or 'end', uuid, sessionID)
```

Comment:
@param and @method not @ param and @ method
Also, usually, @param is followed by the name of the parameter and then the type